### ISSUE TITLE
hikari datsource

### ISSUE THREAD

**Issue by kururu-abdo** [kururu-abdo/InvocationTargetException#3]
how can i use **hikari data source** in intellij idea
is there any plugin that i need  to get start
@edvin

**Comment by edvin** [comment]
Yes you can, there is nothing TornadoFX specific about it and you don't need any plugins. You already found a usage example in TornadoFX issue #24 :)

**Comment by kururu-abdo** [comment]
i get that 
i downloaded hikariCp.jar 
and add it to libraries 
and add mysql driver, too
and i set dependencies in POM file 

and used DB.kt from your isses 

thanks,  for reply @edvin
